### ISSUE TITLE
Russian bots are getting through the validation

### ISSUE THREAD

**Issue by starchild** [jquery-validation/jquery-validation#2007]
Hi,

Despite having this successfully implemented, we have Russian bots submitting form data still and bypassing the validations.

Any ideas to help lock this down would be greatly appreciated. 

Thanks a lot.

**Comment by sevban** [comment]
Are you doing any kind of validation on back-end? Bots can just disable js.

**Comment by CRC-Mismatch** [comment]
They're probably not even bothering with your HTML/JavaScript, other than for having a form template on how to send the data.
JavaScript isn't enforced - in any decent browser you can disable it, so you'll be left with HTML5 form validation that only works in up-to-date browsers (and can also be skipped), and in most of them you can also access the JavaScript console and do whatever you want (also through injected user-scripts - i.e. Tampermonkey).
Also, sending HTTP (GET, POST, etc) requests can be done even through telnet or other raw methods, without any HTML or JavaScript whatsoever getting involved.
Your only hope is to validate data in the back-end.

TL;DR isn't a bug, IMO

**Comment by staabm** [comment]
Will close for now. Feel free to ask further questions regarding this topic here. We can re-open if required.

IMO question was answered above, thank you guys
